scapy/layers/zigbee.py

```diff
@@ -1082,7 +1082,7 @@ class ZigbeeClusterLibrary(Packet):
         # Frame control (8 bits)
         BitField("reserved", 0, 3),
         BitField("disable_default_response", 0, 1),  # 0 default response command will be returned  # noqa: E501
-        BitField("direction", 0, 1),  # 0 command sent from client to server; 1 command sent from server to client  # noqa: E501
+        BitField("command_direction", 0, 1),  # 0 command sent from client to server; 1 command sent from server to client  # noqa: E501
```

Comment:
Could you add a
```python
deprecated_fields = {
    "direction": ("command_direction", "2.5.0"),
}
```
before the `fields_desc` to account for deprecation?

Reply:
Done!